pandas/core/tools/datetimes.py

```diff
@@ -42,22 +42,6 @@ def _lexer_split_from_str(dt_str):
     pass
 
 
-def _infer_tzinfo(start, end):
-    def _infer(a, b):
-        tz = a.tzinfo
-        if b and b.tzinfo:
```

Comment:
where is this actually used?

Reply:
Outside of tests, its used once in `indexes.datetimes`